Kitodo/src/main/java/org/goobi/production/flow/jobs/AbstractGoobiJob.java

```diff
@@ -1,91 +0,0 @@
-/*
- * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
- *
- * This file is part of the Kitodo project.
- *
- * It is licensed under GNU General Public License version 3 or later.
- *
- * For the full copyright and license information, please read the
- * GPL3-License.txt file that was distributed with this source code.
- */
-
-package org.goobi.production.flow.jobs;
-
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.quartz.Job;
```

Comment:
As you remove last usage of `org.quartz` library the dependency in `Kitodo/pom.xml` and file `Kitodo/src/main/resources/quartz.properties` should be removed.

Reply:
Good catch 👍